src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -98,6 +102,10 @@ namespace OpenVic {
 
 		/* Budget */
 		fixed_point_t PROPERTY(cash_stockpile);
+		slider_value_t PROPERTY(poor_tax_rate_slider);
```

Comment:
Sliders are UI elements to adjust the tax rate.
CountryInstance is the simulation data for the country.
CountryInstance needs tax percentage (int). (or a tax rate as fixed point)

Reply:
Nah I think we should include min/max checking in the gamestate itself to be extra safe + provide a single source of truth for slider value clamping. If anything we need another 2 variables in the struct (probably const) representing the final min and max values that the modifier-effect based min and max values are clamped between